docs/howto/configurations-ref.rst

```diff
@@ -0,0 +1,1141 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Configuration Reference
+=======================
+
+.. _config-ref/core:
+
+``[core]``
+^^^^^^^^^^
+
+dags_folder
```

Comment:
Can you also add labels before keys?

Reply:
So add a label for every key in the document? I could probably make a vim shortcut for that but just want to make sure I'm understanding the ask first

Reply:
Yes. I mean creating a label for each key so that we can include a link to each key in the documentation. If you want I will do it very quickly using a regular expression. ;-D

Reply:
Here is  doc with sections: https://pastebin.com/aL7qbni1 (take it or throw it away ;-))